Sqlite/ImportSql.py

```diff
@@ -251,7 +251,7 @@ def get_media_list(self, sql, from_type, from_handle):
         return [self.pack_media_ref(sql, result) for result in results]
 
     def get_surname_list(self, sql, handle):
-        results = sql.query("""select * from surname where handle = ?;""", handle)
+        results = sql.query("""select s.* from surname s inner join link l ON l.to_handle = s.handle where l.from_handle = ?;""", handle)
```

Comment:
Do these need to be ordered? That is, will they be imported/exported and assembled in the correct order?